nexmark-flink/src/main/resources/queries-cep/q0.sql

```diff
@@ -0,0 +1,37 @@
+-- -------------------------------------------------------------------------------------------------
+-- Query 0: Periods of a constantly decreasing price (Not in original suite)
+-- -------------------------------------------------------------------------------------------------
+-- Finds periods of a constantly decreasing price of a single bidder.
```

Comment:
Could you also update the README about the newly added queries?

Reply:
@dianfu, I have discussed with @wuchong offline and will add the document of the MATCH_RECOGNIZE queries in README into another separate PR because the benchmark test result should be updated together.